proto/hierarchies.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+import "matrix.proto";
+
+package bayesmix;
+
+message NNIGPrior {
+  message FixedValues {
+    double mu0 = 1;
+    double lambda0 = 2;
+    double alpha0 = 3;
+    double beta0 = 4;
+  }
+
+  message UnivNormalPrior {
+    double mu00 = 1;
```

Comment:
I agree. Those parameters are first placed into variables such as mu00 when extracted before being used anyways. So it maskes sense to give them generic names in protos and precise names in local variables.

About sub-messages, in fact, there is already a duplicate for the Gamma proto, which is the Gamma proto in mixings. Since we are renaming values with generic names, it makes sense to move them out of the Hyperprior message. At this point, the best thing to do is to create a new "distributions.proto" to hold all these sub-messages so that they can be used everywhere. A reorganization of proto files was long overdue anyways...

I'll fix this later. After that, I'd rather merge this PR and deal with the implementation of the two new hyperpriors you mentioned above in a new PR. What do you think?

Reply:
ok great